### ISSUE TITLE
After creating an account, I was redirected to login, but couldn't login

### ISSUE THREAD

**Issue by adamlofting** [mozilla/id.webmaker.org#201]
1. Testing from http://mozteach.toolness.org/
1. Click Create an Account
1. Fill out form
1. Got redirected to login form [1]
1. Received Welcome to Webmaker Email (which suggests the account is created)
1. Entering my newly created username and password get's me two green validation ticks, but no redirect or indication if I'm actually logged in

cc @toolness incase it's something on the teach.m.o side of this, but it feels like a sticking point on id.w.o

[1] https://id.mofostaging.net/login?client_id=wm_id_1H9nvpGGTQ4CkXGVn56p&response_type=code&state=IpMcypvPtXwtbJ8lIdll6Njb9QLAIWlq&scopes=user%20email

**Comment by cadecairos** [comment]
@adamlofting which web browser & version were you using?

**Comment by adamlofting** [comment]
My default is FF Dev Edition (39). I tried to recreate this now, but wasn't able to create an account this time. Happy to try again.

**Comment by cadecairos** [comment]
Just as I thought! We just discovered why this happened, and fixed it in #225 

Firefox Dev Edition / nightly and Chrome Beta and Canary had implemented the Fetch API [1] natively, while on stable versions of the browsers we used a polyfill. For silly reasons, the polyfill was behaving differently to the native versions of fetch and including credentials by default.

1. https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
